includes/class-wcpdf-frontend.php

```diff
@@ -63,6 +64,21 @@ public function my_account_pdf_link( $actions, $order ) {
 		return apply_filters( 'wpo_wcpdf_myaccount_actions', $actions, $order );
 	}
 
+	/**
+	 * Open PDF on My Account page in a new browser tab/window
+	 */
+	public function open_my_account_pdf_link_on_new_tab() {
+		if ( function_exists( 'is_account_page' ) && is_account_page() ) {
+			if ( $general_settings = get_option( 'wpo_wcpdf_settings_general' ) ) {
+				if ( isset( $general_settings['download_display'] ) && $general_settings['download_display'] == 'display' ) {
+					if ( function_exists( 'file_get_contents' ) && !empty( $file = file_get_contents( WPO_WCPDF()->plugin_url() . '/assets/js/my-account-link.js' ) ) ) {
```

Comment:
This doesn't work with `allow_url_fopen` to false (or other restrictions), and is slow too (loading via the interwebs instead of locally). Using `plugin_path()` instead of `plugin_url()` fixes that
```php
if ( function_exists( 'file_get_contents' ) && $script = file_get_contents( WPO_WCPDF()->plugin_path() . '/assets/js/my-account-link.js' ) ) {
	wp_add_inline_script( 'jquery', $script );
}
```

Reply:
Fixed in last commit.